.travis.yml

```diff
@@ -45,24 +44,8 @@ matrix:
   - node_js: "6"
     env: DISABLE_EMBER_CLI_FEATURES=true; TEST_COMMAND=test-all
 
-before_install:
-  # Once we no longer support Node.js 0.12 we can drop this entire section.
-  # prevent the npm loading indicator
-  - npm config --global set spin false
-  # if npm version is less than 3.0.0, upgrade to 3
-  - if [[ $(npm -v | cut -d '.' -f 1) -lt 3 ]]; then npm i -g npm@^3; fi
-  - if [[ $(npm -v | cut -d '.' -f 1) -gt 4 ]]; then npm i -g npm@^4; fi
-
-  # travis currently includes yarn v0.17.8 (20170705)
-  # this isn't new enough for our use of --non-interactive
-  - curl -o- -L https://yarnpkg.com/install.sh | bash
-  - export PATH=$HOME/.yarn/bin:$PATH
```

Comment:
why did you remove this?

Reply:
I thought Travis comes now with a newer `yarn` version that supports `--non-interactive`. Was I wrong?

Reply:
it's possible, but a) this seems unrelated to the Node 4 changes and b) we might still want to run on the latest yarn version 🤔

Reply:
It seemed not necessary - but if the consensus is that it's needed, I'll bring it back.

Reply:
let's be conservative and remove it in subsequent PR if needed

Reply:
@Turbo87 @twokul Added back